app/services/hyrax/collections/collection_member_search_service.rb

```diff
@@ -46,8 +46,10 @@ def available_member_subcollections
       #
       # @return [Blacklight::Solr::Response]
       def available_member_works
+        sort_field = user_params[:sort]
```

Comment:
Previously the sort params were not getting through, so the sorting selections did nothing.